cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers.go

```diff
@@ -123,23 +180,22 @@ func TryLoadCertAndKeyFromDisk(pkiPath, name string) (*x509.Certificate, *rsa.Pr
 	case *ecdsa.PrivateKey:
```

Comment:
Ok so it is safe to type cast to *rsa.PrivateKey? What if the user generates ECDSA?